src/bodywork/k8s/workflow_jobs.py

```diff
@@ -141,12 +156,12 @@ def configure_workflow_cronjob(
     :return: A configured k8s cronjob object.
     """
     job = configure_workflow_job(
-        namespace=namespace,
-        project_name=project_name,
+        namespace=BODYWORK_DEPLOYMENT_JOBS_NAMESPACE,
```

Comment:
Should we not have `namespace` as an arg to `configure_workflow_cronjob` as well, for the sake of consistency (that being that namespace is an arg at the low-level k8s API and set in stone at the CLI level)?

Reply:
Yes I agree completely, I don't know why I did this one differently.